common-lib/src/main/resources/application.conf

```diff
@@ -23,3 +23,22 @@ image.processors = [
   "com.gu.mediaservice.lib.cleanup.GuardianMetadataCleaners",
   "com.gu.mediaservice.lib.cleanup.SupplierProcessors$"
 ]
+
+authentication.providers {
+  machine {
+    className = "com.gu.mediaservice.lib.auth.provider.ApiKeyAuthenticationProvider"
+    config {
+      # authKeyStoreBucket = <s3-bucket-with-api-keys>
+    }
+  }
+  # TODO: short term we put panda here for backwards compatibility but the default provider should be something better
```

Comment:
Minor point: lets say something like "the default provider should be something that involves zero configuration for a simple test setup", just to help any future archaeologists infer what we mean by "better" here